src/paragon-framework/modules/table/table.scss

```diff
@@ -229,7 +229,7 @@ $flex-table--header-text: $color-white;
 		}
 
 		li {
-			margin-top: 0;
+			margin-top: 0 !important;
```

Comment:
just verify important is the best route

Reply:
I think it is... sadly. Table specificity was overwriting this padding "fix" for loading tables.
So a table structured like
`<div className="flex-table row-selectable is-condensed font-small is-loading" >`
is so specific that I needed to apply an !important to overwrite the margin.